packages/botonic-react/src/components/button.jsx

```diff
@@ -4,7 +4,7 @@ import styled from 'styled-components'
 import { params2queryString, INPUT } from '@botonic/core'
```

Comment:
in the future "the price to pay" to add an util is writing a unit test for them.
this will ensure that we don't create too many :-)

Reply:
sure, will add them in next PRs 👍